NEMO/views/notifications.py

```diff
@@ -84,6 +85,21 @@ def create_buddy_request_notification(buddy_request: BuddyRequest):
             )
 
 
+def create_staff_assistance_request_notification(staff_assistance_request: StaffAssistanceRequest):
+    users: List[User] = User.objects.filter(is_active=True).exclude(id=staff_assistance_request.user_id)
```

Comment:
you should limit to staff so other people don't get notified
```suggestion
    users: List[User] = User.objects.filter(is_active=True, is_staff=True).exclude(id=staff_assistance_request.user_id)
```